src/xmipp/tests/test_programs_xmipp.py

```diff
@@ -1530,3 +1530,12 @@ def test_case1(self):
                 outputs=["test1/iter22_img.xmd","test1/iter22_it000001_ref.xmd","test1/iter22_ref.xmd"])
 
 
+class VolSubtraction(XmippProgramTest):
+    _owner = DISCONTINUED
+    @classmethod
+    def getProgram(cls):
+        return 'xmipp_volume_subtraction'
+
+    def test_case1(self):
+        self.runCase("--i1 input/phantomBacteriorhodopsin.vol --i2 input/phantomBacteriorhodopsin.vol -o %o/output_volume.mrc --iter 5 --lambda 1.0 --sub --cutFreq 1.333333 --sigma 3 --radavg --computeEnergy",
+                outputs=["output_volume.mrc"])
```

Comment:
this only tests that some output is produced, right? Please check also that the content of the file is correct